lib/improver/tests/nowcasting/utilities/test_ApplyOrographicEnhancement.py

```diff
@@ -32,36 +32,36 @@
 """Module with tests for the ApplyOrographicEnhancement plugin."""
 
 import unittest
-
+import numpy as np
 from cf_units import Unit
+from datetime import datetime
+
 import iris
 from iris.tests import IrisTest
-import numpy as np
 
 from improver.nowcasting.utilities import ApplyOrographicEnhancement
-from improver.tests.ensemble_calibration.ensemble_calibration.\
-    helper_functions import set_up_cube
+from improver.tests.set_up_test_cubes import set_up_variable_cube
 
 MIN_PRECIP_RATE_MMH = ApplyOrographicEnhancement("add").min_precip_rate_mmh
 
 
-def set_up_precipitation_rate_cube():
+def set_up_precipitation_rate_cubelist():
```

Comment:
If we don't need any more we can remove a line from the plugin supporting this functionality: https://github.com/metoppv/improver/blob/master/lib/improver/nowcasting/utilities.py#L177

Reply:
@fionaRust I'm guessing you're referring to the discussion below?  In which case you're probably right, I could remove this, but I'm trying not to touch functionality when updating the unit tests just in case...